### PR TITLE
Story/#132 support the RIF 1.2 format

### PR DESCRIPTION
**Pull request by sascha15432**

### CODE REVIEW

File: src/main/java/org/itsallcode/openfasttrace/importer/rif/handler/RifObjectsHandlerBuilder.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                .artifactType(this.defaultDoctype);
+        this.handler.pushDelegate(new SingleRifObjectsHandlerBuilder(this.listener, this.idBuilder,
+                this.locationBuilder).build());
+    }
+
+    private void handleEndElement()
+    {
+        this.listener.setId(this.idBuilder.build());
+        this.listener.endSpecificationItem();
+        this.listener.setLocation(this.locationBuilder.build());
```
Comment (kaklakariada) [inline src/main/java/org/itsallcode/openfasttrace/importer/rif/handler/RifObjectsHandlerBuilder.java]: Not sure, but i think `setLocation()` should be called *before* `endSpecificationItem()`

File: src/main/java/org/itsallcode/openfasttrace/importer/rif/handler/SingleRifObjectsHandlerBuilder.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    public CallbackContentHandler build()
+    {
+        configureDataHandlers();
+        configureSubTreeHanlders();
+        ignoreCharacterData("creationdate", "source");
+        return this.handler;
+    }
+
+    private void configureSubTreeHanlders()
```
Comment (kaklakariada) [inline src/main/java/org/itsallcode/openfasttrace/importer/rif/handler/SingleRifObjectsHandlerBuilder.java]: Typo in method name

File: src/main/java/org/itsallcode/openfasttrace/importer/rif/handler/SingleRifObjectsHandlerBuilder.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                .addCharacterDataListener("sourcefile", this::rememberSourceFile)
+                .addIntDataListener("sourceline", this::rememberSourceLine);
+    }
+
+    private void setStatus(final String statusAsText)
+    {
+        this.listener.setStatus(ItemStatus.parseString(statusAsText));
+    }
+
+    private void removeArtifactTypeFromName(final String data)
```
Comment (kaklakariada) [inline src/main/java/org/itsallcode/openfasttrace/importer/rif/handler/SingleRifObjectsHandlerBuilder.java]: Maybe rename method to `setIdName()`

**Comment by redcatbear** [comment]
@sascha15432 are you still planning to go ahead with this PR or should we close it?

**Pull request closed without merging**